apps/ws/src/bootstrap.ts

```diff
@@ -18,9 +19,14 @@ if (process.env.SENTRY_DSN) {
 }
 
 export async function bootstrap() {
-  const app = await NestFactory.create(AppModule);
+  const app = await NestFactory.create(AppModule, { bufferLogs: true });
   const redisIoAdapter = new RedisIoAdapter(app);
 
+  app.useLogger(app.get(Logger));
+  app.flushLogs();
+
+  app.useGlobalInterceptors(getErrorInterceptor());
```

Comment:
Is this realy needed when using pino nestjs?

Reply:
Yes, because we are using an async logger we need to make sure it is up and running so we dont lose any logs when the system starts up.

Reply:
Was not referring to flushLogs, mainly to the `useGlobalInterceptor`, from what I recall nest-pino adds one by default when autoLogging is enabled. (sorry for the confusion)

Reply:
Autologging is only for req/res not capturing errors, that is where the ErrorInterceptor comes in, thus this is needed to capture errors.